### PR TITLE
Updates on some of the repo files

### PR DESCRIPTION
**Pull request by CoderBlee**

### CODE REVIEW

**Comment by ronnzw** [comment]
@CoderBlee Thank you for your hard work documenting our processes. Can you fix the conflict please.

File: .github/ISSUE_TEMPLATE/task.md [hunk 1]
```diff
@@ -1,14 +1,95 @@
...
+## Venue:
+- Location: [Choose a Venue or Virtual Platform]
+- Address: [Physical Address if in-person, or platform details if virtual]
 
-Due Date: %date
+## Tasks - Do these in Order
 
-Description:
+### Objective:
+To bring together Python, hackerthors and absolute beginners enthusiasts to collaborate on open-source projects, learn new skills, and network with like-minded individuals.
```
Comment (ronnzw) [inline .github/ISSUE_TEMPLATE/task.md:19]:
```suggestion
To bring Python enthusiasts, hackathon participants, and absolute beginners in collaborative open-source projects, fostering skill development and creating opportunities for networking with like-minded individuals.
```

Comment (ronnzw) [reply .github/ISSUE_TEMPLATE/task.md:19]: Rewording to bring your point home.

File: .github/ISSUE_TEMPLATE/task.md [hunk 2]
```diff
@@ -1,14 +1,95 @@
...
+   - **Details:** Summarize the day’s achievements, thank all participants and volunteers. Award small prizes for categories like “Most Innovative Project,” “Best Collaboration,” etc.
+
+**9. After-Party (Optional)**
+   - **Time:** 4:30 PM – 6:00 PM
+   - **Details:** Continue networking and relaxing. Offer refreshments and perhaps a casual discussion or Q&A session with the keynote speaker.
+
+### Team Activities:
+
+1. **Mini Workshops:**
+   - **“Advanced Python Techniques”** – A workshop focusing on more complex aspects of Python. If possible also sell the BPD merch
```
Comment (ronnzw) [inline .github/ISSUE_TEMPLATE/task.md:63]:
```suggestion
   - **“Advanced Python Techniques”** – A workshop focusing on more complex aspects of Python. If possible also sell the BPD merchandise.
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 3]
```diff
@@ -1,14 +1,95 @@
...
+   - **“Open Source Best Practices”** – Tips and strategies for contributing to open-source projects.
+
+2. **Creative Coding Challenges:**
+   - Short coding challenges or puzzles related to the day’s projects. Offer small rewards for those who solve them.
+
+3. **Mentorship Circles:**
+   - Form small groups where experienced developers mentor beginners. This fosters knowledge sharing and builds community.
```
Comment (ronnzw) [inline .github/ISSUE_TEMPLATE/task.md:70]:
```suggestion
   - Form small groups where experienced developers mentor beginners. This fosters knowledge sharing and builds the community.
```

File: .github/ISSUE_TEMPLATE/task.md [hunk 4]
```diff
@@ -1,14 +1,95 @@
...
+
+### Promotion:
+
+1. **Social Media:**
+   - Create event pages on platforms like Twitter, LinkedIn, and Facebook. Use hashtags relevant to Python and open-source communities.
```
Comment (ronnzw) [inline .github/ISSUE_TEMPLATE/task.md:75]:
```suggestion
   - Create event pages on platforms like Twitter( X), LinkedIn, and Facebook. Use hashtags relevant to Python and open-source communities.
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,12 @@
 # The Black Python Devs Maintainers Guide
 
-We want to ensure that our community is transparent in how we think, act, and respond. For this reason we have opensourced our community guides.
+Welcome to the Black Python Devs Maintainers Guide! This guide is designed to ensure transparency and consistency in how we, as a community, think, act, and respond. By open-sourcing our community guides, we aim to foster collaboration, inclusivity, and accountability.
 
-This repo entails the programs, policies, roles, and responsibilities of the community. 
+This repository outlines the **programs**, **policies**, **roles**, and **Initiatives** that shape our community. Whether you're a current maintainer, a **contributor**, or someone looking to understand how we work, this guide provides everything you need to know.
```
Comment (ronnzw) [inline README.md:5]: I suggest adding links to the bolded elements for easy access by readers.

File: how_to/contribute_to_the_bpd_community.md [hunk 6]
```diff
@@ -1 +1,87 @@
 # How to Contribute to the BPD Community
+Contributing to open source can be a rewarding experience, and getting started is easier than you might think. Here’s a beginner-friendly guide to help you navigate your way into the open source world for Black Python Dev repo:
```
Comment (ronnzw) [inline how_to/contribute_to_the_bpd_community.md:2]:
```suggestion
Contributing to open source can be a rewarding experience, and getting started is easier than you might think. Here’s a beginner-friendly guide to help you navigate your way into the open source world for Black Python Devs repo:
```

File: how_to/contribute_to_the_bpd_community.md [hunk 7]
```diff
@@ -1 +1,87 @@
...
+Open source software is developed collaboratively and made available to anyone to use, modify, and distribute. Contributions can range from fixing bugs and adding features to improving documentation.
+
+### 2. **Find a Project**
+
+
+
+#### **Look for Good First Issues**
+
+Many projects label issues that are suitable for beginners. Search for labels like “good first issue,” “beginner,” or “help wanted.”
+You can use this link to see some of the available issues: https://github.com/BlackPythonDevs/blackpythondevs/issues
```
Comment (ronnzw) [inline how_to/contribute_to_the_bpd_community.md:16]:
```suggestion
You can use [this] (https://github.com/BlackPythonDevs/blackpythondevs/issues) link to see some of the available issues.
```

File: how_to/contribute_to_the_bpd_community.md [hunk 8]
```diff
@@ -1 +1,87 @@
...
+
+
+#### **Read the Documentation**
+
+- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the read me first:
```
Comment (ronnzw) [inline how_to/contribute_to_the_bpd_community.md:21]:
```suggestion
- **README.md**: The primary document that usually provides an overview, installation instructions, and usage guidelines.  Read the README first:
```

**Review by ronnzw — COMMENTED** [review]
This is well documented thank you for taking the time. I have commented on a few gram related items, that l have provided suggestions. Also commented on are typos. Generally this is very well done. I have asked @kjaymiller to review the content but generally l'm happy. Thank you so much.

File: .github/ISSUE_TEMPLATE/task.md (hunk 1 above)
Comment (CoderBlee) [reply .github/ISSUE_TEMPLATE/task.md:19]: Thank you soo much for looking into it. I was discouraged and honestly did not know what to do anymore. I also learned not to contribute in bulk

File: how_to/contribute_to_the_bpd_community.md (hunk 8 above)
Comment (CoderBlee) [reply how_to/contribute_to_the_bpd_community.md:21]: Does it mean that it will be merged now? since you have fixed some of the typos

Comment (ronnzw) [reply how_to/contribute_to_the_bpd_community.md:21]: @CoderBlee if you are happy with the suggestions you can commit them and let us know.

Comment (CoderBlee) [reply how_to/contribute_to_the_bpd_community.md:21]: @ronnzw  the one with conflicts, I do not know how to solve it anymore, also how do I save the new changes without is being flagged

**Comment by kjaymiller** [comment]
@CoderBlee - 

I'm sorry but we need to close this issue. Here are the following reasons:

- There is no single issue that this project resolves.
- The templates have personal information in them.
- The changes to policy and procedure were not approved by the council or leadership team.

Please reach out to us on our discord. We appreciate the effort that you put into this and we want to help you get a PR pushed. We hope you will continue working with us.

**Pull request closed without merging**